src/io/flutter/logging/FlutterLogConstants.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.logging;
+
+import com.intellij.execution.ui.ConsoleViewContentType;
+import com.intellij.openapi.editor.colors.TextAttributesKey;
+import com.intellij.openapi.util.Key;
+import org.jetbrains.annotations.NotNull;
+
+import static com.intellij.execution.ui.ConsoleViewContentType.registerNewConsoleViewType;
+
+public final class FlutterLogConstants {
+  private FlutterLogConstants() {
+  }
+
+  @NotNull
+  public static final TextAttributesKey NONE_OUTPUT_KEY =
+    TextAttributesKey.createTextAttributesKey("LOGCAT_NONE_OUTPUT", ConsoleViewContentType.NORMAL_OUTPUT_KEY);
```

Comment:
I think we want to change all of these keys to `"FLUTTER_LOG_*`?  So, for example: `"FLUTTER_LOG_NONE_OUTPUT"`?

Reply:
Yeap, I updated.